labs/src/test/scala/org/scalalabs/basic/lab02/CollectionExerciseTest.scala

```diff
@@ -37,7 +37,7 @@ class CollectionExerciseTest extends Specification {
       val erik = new Person(24, "Erik")
       val susy = new Person(40, "Susy")
 
-      val result = CollectionExercise02.groupAdultsPerAgeGroup(Seq(jack, duke, jeniffer, erik, susy))
+      val result = CollectionExercise02.groupAdultsPerAgeGroup(Seq(jack, jeniffer, duke, erik, susy))
```

Comment:
a long time since I did that, but I suppose that to test that one does a 'sort' we should provide a different order as an input.
otherwise an implementation could ignore sorting and the test would pass. this enforces implementation to really do sorting here